src/backends/github/API.js

```diff
@@ -329,37 +329,126 @@ export default class API {
     } else {
       // Entry is already on editorial review workflow - just update metadata and commit to existing branch
       return this.getBranch(branchName)
-      .then(branchData => this.updateTree(branchData.commit.sha, "/", fileTree))
-      .then(changeTree => this.commit(options.commitMessage, changeTree))
-      .then((response) => {
-        const contentKey = entry.slug;
-        const branchName = `cms/${ contentKey }`;
```

Comment:
These two lines are just duplicating/shadowing the `contentKey`/`branchName` declarations at the top of the function.